README.md

```diff
@@ -1,3 +1,17 @@
+# Infomed
+
+Web portal that enables communication between hospitals and health authorities
+
+# Contributing
+
+1. Go to https://github.com/Lifespark-Technologies/Infomed/projects/1
+2. Pick an unassigned card from the "To do" column and assign it to yourself. If you want to work on something that is assigned to someone else, talk to that person (or @bl-nero) first.
+3. As soon as you start working on an issue, move it to the "In progress" column. Don't assign yourself to tasks that are in progress, but also don't hold on to a task that you can't contribute to in a reasonable timeframe.
+4. Request a code review. Don't merge code without it. Don't push code directly to `master`.
+5. Merge to `master`!
+
```

Comment:
A possible addition to 3: maybe include a rough timeline for when you expect to finish the issue?

Reply:
I wanted to deliberately avoid being too specific here, since every case will be different, depending on priority, complexity, and what's blocked by given issue.

Reply:
Ah gotcha, that sounds reasonable then.